prime-router/src/main/java/fhirengine/translation/hl7/structures/fhirinventory/group/ORM_O01_PATIENT.java

```diff
@@ -0,0 +1,328 @@
+ * specific language governing rights and limitations under the License.
+ *
+ * The Original Code is "[file_name]".  Description:
+ * "[one_line_description]"
+ *
+ * The Initial Developer of the Original Code is University Health Network. Copyright (C)
+ * 2012.  All Rights Reserved.
+ *
+ * Contributor(s): ______________________________________.
+ *
+ * Alternatively, the contents of this file may be used under the terms of the
+ * GNU General Public License (the  "GPL"), in which case the provisions of the GPL are
+ * applicable instead of those above.  If you wish to allow use of your version of this
+ * file only under the terms of the GPL and not to allow others to use your version
+ * of this file under the MPL, indicate your decision by deleting  the provisions above
+ * and replace  them with the notice and other provisions required by the GPL License.
+ * If you do not delete the provisions above, a recipient may use your version of
+ * this file under either the MPL or the GPL.
+ *
+ */
+
+package fhirengine.translation.hl7.structures.fhirinventory.group;
+
+import ca.uhn.hl7v2.model.v27.segment.*;
+
+import ca.uhn.hl7v2.HL7Exception;
+import ca.uhn.hl7v2.parser.ModelClassFactory;
+import ca.uhn.hl7v2.model.*;
+import ca.uhn.hl7v2.model.v26.group.ORM_O01_INSURANCE;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.NTE;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.PD1;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.PID;
+
+/**
+ * <p>Represents a ORM_O01_PATIENT group structure (a Group object).
+ * A Group is an ordered collection of message segments that can repeat together or be optionally in/excluded together.
+ * This Group contains the following elements:
+ * </p>
+ * <ul>
+ * <li>1: PID (Patient Identification) <b>  </b></li>
+ * <li>2: PD1 (Patient Additional Demographic) <b>optional  </b></li>
+ * <li>3: NTE (Notes and Comments) <b>optional repeating </b></li>
+ * <li>4: ORM_O01_PATIENT_VISIT (a Group object) <b>optional  </b></li>
+ * <li>5: ORM_O01_INSURANCE (a Group object) <b>optional repeating </b></li>
+ * <li>6: GT1 (Guarantor) <b>optional  </b></li>
+ * <li>7: AL1 (Patient Allergy Information) <b>optional repeating </b></li>
+ * </ul>
+ */
+//@SuppressWarnings("unused")
+public class ORM_O01_PATIENT extends AbstractGroup {
+
+  /**
+   * Creates a new ORM_O01_PATIENT group
+   */
+  public ORM_O01_PATIENT(Group parent, ModelClassFactory factory) {
+    super(parent, factory);
+    init(factory);
+  }
+
+  private void init(ModelClassFactory factory) {
+    try {
+      this.add(PID.class, true, false, false);
+      this.add(PD1.class, false, false, false);
+      this.add(NTE.class, false, true, false);
+      this.add(ORM_O01_PATIENT_VISIT.class, false, false, false);
+      // If IN1 related classes are built out and ORM_O01_INSURANCE replaced, all hapiv26 dependencies can be removed
```

Comment:
Makes sense, I think there is a ticket Arnej wrote (will write?), could we tag it here?

Reply:
Also, just to confirm, the mappings for IN1 still need to be built right?

Reply:
Talked to Arnej earlier this week and it sounds like building out more java classes is likely to be completed before the IN1 work. I linked [that ticket](https://github.com/CDCgov/prime-reportstream/issues/16307) in the comment and am leaving comments there and on the ticket for building out INx hapi classes.

Reply:
[Added Comment](https://github.com/CDCgov/prime-reportstream/pull/16299/commits/45354ae50d215895943668c32bdcc50d1106f4bd)